charts/lighthouse-ci/values.yaml

```diff
@@ -164,3 +164,9 @@ basicAuthPassword: ""
 
 # Name of existing Secret to use
 existingSecret: ""
+
+env: {}
```

Comment:
As I'm trying to mimic Bitnami charts, please rename the variable to  `extraEnvVars` and add a comment like `Extra environment variables` before the value. Fix also the README.md with this new variable name, the comment and the default `nil`value.